frontend/src/components/OmakaseModal.jsx

```diff
@@ -0,0 +1,209 @@
+    setFormData((prevFormData) => {
+      const newOperation = e.target.value;
+      if (
+        prevFormData.operation === "Segment Concatenation" &&
+        newOperation !== "Segment Concatenation"
+      ) {
+        return {
+          ...prevFormData,
+          operation: newOperation,
+          filename: "",
+          bucket: "",
+          path: "",
+        };
+      }
+      return { ...prevFormData, operation: newOperation };
+    });
+  };
+
+  return (
+    <>
+      <Modal
+        onDismiss={() => setOmakaseModalVisible(false)}
+        visible={omakaseModalVisible}
+        header="Export"
+      >
+        <SpaceBetween direction="vertical" size="l">
+          <FormField label="Operation">
+            <Select
+              selectedOption={{
+                label: formData.operation,
+                value: formData.operation,
+              }}
+              onChange={({ detail }) => {
+                const e = { target: { value: detail.selectedOption.value } };
+                handleOperationChange(e);
+              }}
+              options={operations.map((op) => ({ label: op, value: op }))}
+            />
+          </FormField>
+
+          {formData.operation === "Segment Concatenation" && (
+            <FormField label="Format">
+              <Select
+                selectedOption={{
+                  label: formData.format,
+                  value: formData.format,
+                }}
+                onChange={({ detail }) =>
+                  setFormData((prevFormData) => ({
+                    ...prevFormData,
+                    format: detail.selectedOption.value,
+                  }))
+                }
+                options={formats.map((format) => ({
+                  label: format,
+                  value: format,
+                }))}
+              />
+            </FormField>
+          )}
+
+          {formData.operation === "Flow Creation" && (
+            <FormField label="Label">
+              <Input
+                value={formData.label}
+                onChange={({ detail }) =>
+                  setFormData({ ...formData, label: detail.value })
+                }
+                placeholder="Label"
+              />
+            </FormField>
+          )}
+
+          <SpaceBetween direction="vertical" size="s">
+            {flows
+              .filter((flow) => flow.format === "urn:x-nmos:format:audio")
+              .map((flow) => (
+                <Checkbox
+                  key={flow.id}
+                  checked={formData.flows[flow.id] || false}
+                  onChange={({ detail }) =>
+                    setFormData((prev) => ({
+                      ...prev,
+                      flows: { ...prev.flows, [flow.id]: detail.checked },
+                    }))
+                  }
+                >
+                  {flow.description ?? ""}
+                </Checkbox>
+              ))}
+          </SpaceBetween>
+
+          <ExpandableSection
```

Comment:
Expandandable section and fields should only be present for Segment Concatentation. In the original they "disabled" the advanced part but for us i think we will just need to use conditional render?